discovery-provider/plugins/notifications/src/processNotifications/mappers/supportingRankUp.ts

```diff
@@ -96,14 +104,24 @@ export class SupportingRankUp extends BaseNotification<SupportingRankUpNotificat
       )
       await this.incrementBadgeCount(this.senderUserId)
     }
-    //
 
     if (
+      isLiveEmailEnabled &&
       userNotificationSettings.shouldSendEmail({
         receiverUserId: this.senderUserId
       })
     ) {
-      // TODO: Send out email
+      const notification: AppEmailNotification = {
+        receiver_user_id: this.receiverUserId,
```

Comment:
again here, the receiver of this notification is actually the person sending the tip, so it should be `this.senderUserId`. we should definitely rename these to `this.tipSenderUserId` or something 🤦‍♀️

Reply:
same for line 119